cmd/serf/command/agent/command.go

```diff
@@ -92,11 +92,22 @@ func (c *Command) readConfig() *Config {
 	cmdFlags.StringVar(&retryInterval, "retry-interval", "", "retry join interval")
 	cmdFlags.BoolVar(&cmdConfig.RejoinAfterLeave, "rejoin", false,
 		"enable re-joining after a previous leave")
+
+	var disableCompression bool
```

Comment:
_super_ nitpicky but all the other free variables used for reading options are defined up at the start of this method. I don't have a strong preference either way and can see benefits to doing it here but I feel like being consistent with existing code is probably more important than any other potential benefits!